metacat-main/src/main/java/com/netflix/metacat/main/services/search/ElasticSearchEventHandlers.java

```diff
@@ -250,9 +249,9 @@ private void updateEntitiesWithSameUri(final String metadataType, final TableDto
             final List<String> ids = es.getTableIdsByUri(metadataType, dto.getDataUri());
             ids.remove(dto.getName().toString());
             if (!ids.isEmpty()) {
-                final ObjectNode node = metacatJsonLocator.emptyObjectNode();
-                node.set("dataMetadata", dto.getDataMetadata());
-                es.updates(ElasticSearchDoc.Type.table.name(), ids, metacatRequestContext, node);
+               final ElasticSearchDoc doc = new ElasticSearchDoc(dto.getName().toString(), dto,
```

Comment:
Is this correct?

Reply:
I am not sure what the previous code is doing to be honest. In the new world, the API does not take in the node, we can talk about it and update.

Reply:
Updated the logic to previous implementation.